nrf_security/src/zephyr/CMakeLists.txt

```diff
@@ -33,9 +35,12 @@ if(CONFIG_BUILD_WITH_TFM)
     )
   endif()
 
+  # For some reason $<TARGET_PROPERTY:tfm,TFM_BINARY_DIR> does not work here
```

Comment:
Probably due to this problem:
https://github.com/zephyrproject-rtos/zephyr/pull/41923/commits/14279d10e89abad9574929a0b710c3ab0ed90dff